pkgs/development/libraries/tracker/default.nix

```diff
@@ -29,13 +29,13 @@
 
 stdenv.mkDerivation rec {
   pname = "tracker";
-  version = "3.4.0";
+  version = "3.4.1";
```

Comment:
This is already on staging: https://github.com/NixOS/nixpkgs/blob/staging/pkgs/development/libraries/tracker/default.nix

Reply:
removed the commit